examples/ex_ft_buffer/ex_ft_buffer.pro

```diff
@@ -0,0 +1,138 @@
+#--------------------------------------------------------------------------------------------------------------
```

Comment:
Can you briefly elaborate if it makes sense to keep the example? Does it use the same code as the ft plugin in MNE Scan?

Reply:
It does not. I meant to delete it but forgot to. Newest push will remove it.